tests/monitoring/test_evaluators.py

```diff
@@ -1,6 +1,7 @@
 import numpy
 
 import theano
+from nose.tools import assert_raises
```

Comment:
Maybe just import from `numpy.testing` instead... Bit silly maybe, because `numpy.testing` is mostly just a redirect to `nose.tools`, but we might as well keep things simple by always using `numpy.testing` for our test suites, instead of mixing `numpy.testing`, `nose.tools`, `nose2`, `unittest2`... Thinking about it, testing is a mess!